### ISSUE TITLE
Table hardly work with screen reader

### ISSUE THREAD

**Issue by 42tte** [SpareBank1/designsystem#502]
#### Which package is this issue related to?
ffe-tables-react

#### Describe your issue (screenshots welcome!)
The table does not work with screen reader (VO/Safari) and needs thorough testing and overhauling. Exemple of bugs:
* No information on sorting, which way and if it's already sorted
* tr with role=button or role=presentation beats the purpose of using a table
* 

#### What is the expected behavior? 
It should be usable with screen reader

#### What is the actual behavior?
It is not

#### Steps to reproduce
Use a screen reader on https://design.sparebank1.no/styleguidist/index.html#!/Table

**Comment by selbekk** [comment]
Are there any prior art out there on developing tables with expandable rows?

**Comment by ksawerysb1** [comment]
Expandable rows are used in many places. Also there are tables with clickable rows or cells.
Would be great to have good a11y guidelines on those use cases.

**Comment by selbekk** [comment]
I asked about expandable rows on [StackOverflow](https://stackoverflow.com/questions/53314689/make-expandable-table-rows-accessible) - perhaps we can get some good links from there.
